PYME/LMVis/Extras/animation.py

```diff
@@ -150,11 +189,16 @@ def run(self, event):
         self.play(False)
 
     def play(self, save):
-        width = self.get_canvas().Size[0]
-        height = self.get_canvas().Size[1]
+        # Fail gracefully if there are no snapshots
+        if len(self.snapshots) == 0:
+            dlg = wx.MessageDialog(None, 'Need views to generate animation! Please create some by pressing `Add` in the animation pane.', 'No animation views')

```

Comment:
- Missing a `dlg.Destroy()`
- For future reference, `wx.MessageBox()` does this all for you (roughly equiv to  `dlg = wx.MessageDialog(); slg.ShowModal(); dlg.Destroy()`)